runtime/src/main/kotlin/kotlin/collections/Arrays.kt

```diff
@@ -16,7 +16,7 @@ import kotlin.util.sortArray
 public actual inline fun <reified T> Array<out T>?.orEmpty(): Array<out T> = this ?: emptyArray<T>()
 
 
-@PublishedApi internal inline fun checkCopyOfRangeArguments(fromIndex: Int, toIndex: Int, size: Int) {
```

Comment:
Not sure if removing inline here is a good idea.

Reply:
Yes, it should not be removed. But I found that kotlin-compiler issues a warning here stating that inlining in this case makes performance impact insignificant.